build/gen_dev_docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR_UUID=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+user=$(id -un)
+group=$(id -gn)
+uid=$(id -u)
+gid=$(id -g)
+tmpdir="/tmp/fdb-docker-${DIR_UUID}"
+image=fdb-dev
+
+pushd .
+mkdir ${tmpdir}
+cd ${tmpdir}
+
+cat <<EOF >> Dockerfile
+FROM foundationdb/foundationdb-build:latest
+RUN groupadd -g ${gid} ${group} && useradd -u ${uid} -g ${gid} -m ${user}
+
+USER ${user}
+CMD scl enable devtoolset-8 rh-python36 rh-ruby24 -- bash
+
+EOF
+
+echo "Created ${tmpdir}"
+echo "Buidling Docker container ${image}"
+sudo docker build -t ${image} .
```

Comment:
You can locally add yourself to your docker group so that you don’t have to sudo your docker commands

Reply:
I wanted to have something that will work for most users but that people can edit later. Using `sudo` seems to be the thing that is most portable.

Alternatively this script would need to:
1. Check if group docker exists
  a. if not - create docker group and
  b. restart docker daemon
1. add user to docker group

Alternatively we will just have another tutorial "how do I need to set up docker" that people will ignore

Reply:
Ok - I am now adding the user to the group `docker` and ask her to log out and back in